assets/js/boldgrid-inspirations.js

```diff
@@ -1554,6 +1544,24 @@ IMHWPB.InspirationsDesignFirst = function( $, configs ) {
 		);
 	};
 
+	/**
+	 * Determines whether or not the category for a build is hidden
+	 * from the sidebar.
+	 *
+	 * @since 2.7.3
+	 *
+	 * @param {object} build The build to check.
+	 *
+	 * @returns {boolean} True if the category is hidden, false otherwise.
+	 */
+	this.getCategoryHidden = function( build ) {
+		if ( self.categories[ build.ParentCategoryId ] && self.categories[ build.ParentCategoryId ].subcategories[build.CategoryId] ) {
```

Comment:
Why is this check needed? In what scenario will the parent category or sub category not exist?

Reply:
I don't know if that scenario ever will exist... I just thought it'd be safest to check, just in case that scenario ever does arise, we don't break something.